skins/kdark/css/layout.css

```diff
@@ -635,7 +635,7 @@ body {
 .volumeControl .ui-slider-horizontal.slider {
 	display: inline-block;
 	width: 0;
-	background: #7c7c7c;
+	background: #606060;
```

Comment:
did you make sure this color is aligned with color scheme?

Reply:
@OrenMe  where is the color scheme ?

Reply:
It's all over the layout.css file.
Why did you decide to change the value to this?

Reply:
@OrenMe  because we got a reject from our accessibility expert - the colors do not fulfill color-contrast requirements. 
There is a very old story here: 
https://kaltura.atlassian.net/browse/FEC-2297?jql=text%20~%20%22contrast%22 
We implemented this: 
https://kaltura.atlassian.net/browse/FEV-43?focusedCommentId=312712&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-312712